packages/drivers/odsp-driver/src/epochTracker.ts

```diff
@@ -550,6 +561,7 @@ export class EpochTrackerWithRedemption extends EpochTracker {
 				addInBody,
 				fetchReason,
 			);
+			// eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Seems like we could do the same `Partial<>` casts from other places in the PR and make this `unknown`.

Reply:
I'm not sure what type `statusCode` is supposed to be coming from. To be safe, I left it alone.